integrations-and-supported-tools/fastai/notebooks/Neptune_fastai.ipynb

```diff
@@ -362,7 +358,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.8"
+   "version": "3.8.15 | packaged by conda-forge | (default, Nov 22 2022, 08:42:03) [MSC v.1929 64 bit (AMD64)]"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "a9715cf0b0024f6e1c62cb31a4f1f43970eb41991212681878768b4bfe53050a"
+   }
```

Comment:
I think the notebook wasn't stripped of its outputs.

Reply:
I think this is something new from VScode, hadn't seen his before.
Will see if I can strip it. Not a problem either way though